coremltools/models/model.py

```diff
@@ -121,6 +123,9 @@ class NeuralNetworkShaper(object):
 
     def __init__(self, model, useInputAndOutputShapes=True):
 
+        msg = "Class {} will be deprecated from version 4"
```

Comment:
I would rewrite this message in a similar way to the other message.

Reply:
Done